content/api/webhooks.apib

```diff
@@ -63,14 +90,36 @@ List an example of the event data that will be posted by a Webhook for the speci
             Accept: application/json
 
 
-## Create [/webhooks]
++ Response
+    
+    {
+      "results": [
+        {
+          "msys": {
+            "message_event": { ... }
+          }
+        }
+      ]
+    }
+
+
+### Create a Webhook [POST /webhooks]
 
-### Create a Webhook [POST]
+When a webhook is created, a test POST request is sent to the `target` URL. If this request does not receive an HTTP 200 response, your request to the Webhook API will fail with HTTP 400 and the webhook will not be created. If created successfully, the webhook will begin to receive event data after 1 minute.
 
-Create a webhook by providing a **webhooks object** as the POST request body.  On creation, events will
-begin to be pushed to the target URL specified in the POST request body.
++ Data Structure
+    + name (string, required) - User-friendly name for webhook
+    + target (string, required) - URL of the target to which to POST event batches. 
+    + events (array, required)  - Array of event types this webhook will receive. Use the [Events Documentation](#webhooks-get-events-documentation-get) endpoint to list the available event types.
```

Comment:
Not important, but the webhook will "send/post" these events, not "receive" them right (ignoring backend infra)?